src/utils/operator.ts

```diff
@@ -40,13 +40,23 @@ function isOrderPartiallyFilled(order: RawOrder): boolean {
   }
 }
 
+export interface RawOrderWithStatus extends RawOrder {
+  status: RawOrderStatusFromAPI
```

Comment:
Do you need to extend it?
Would it work if you added `status` to `RawOrder`?

Reply:
I've added it, I wasn't sure if `RawOrder` could have `status` or not.
Since all the logic and tests don't consider it.